app/controllers/users_controller.rb

```diff
@@ -12,6 +12,10 @@ def create
 		end
 	end
 
+	def show
+		@user = current_user
```

Comment:
I'll suggest you use `@user = User.find(params[:id])` here because the current user could possibly be an admin.